rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -0,0 +1,222 @@
+from werkzeug.exceptions import BadRequest
+
+from .participant_summary_dao import ParticipantSummaryDao
+from model.participant_summary import ParticipantSummary
+from participant_enums import EnrollmentStatus, TEST_HPO_NAME, TEST_EMAIL_PATTERN
+from dao.hpo_dao import HPODao
+
+class ParticipantCountsOverTimeService(ParticipantSummaryDao):
+
+  def __init__(self):
+    super(ParticipantSummaryDao, self).__init__(ParticipantSummary)
+
+  def get_filtered_results(self, start_date, end_date, filters, stratification='ENROLLMENT_STATUS'):
+    """Queries DB, returns results in format consumed by front-end
+
+    :param start_date: Start date object
+    :param end_date: End date object
+    :param filters: Objects representing filters specified in UI
+    :param stratification: How to stratify (layer) results, as in a stacked bar chart
+    :return: Filtered, stratified results by date
+    """
+
+    self.test_hpo_id = HPODao().get_by_name(TEST_HPO_NAME).hpoId
+    self.test_email_pattern = TEST_EMAIL_PATTERN
+
+    # Filters for participant_summary (ps) and participant (p) table
+    filters_sql_ps = self.get_facets_sql(filters)
```

Comment:
Switch back to just calling this once, drop table_prefix as a parameter, and have the facet map contain the table prefix for each column as mentioned above.